Documentation/building/unix/dependencies.md

```diff
@@ -129,10 +129,27 @@ of a program within `$PATH` during the build. If a required program doesn't
 exist, then the build will fail and a suggested `brew install` command line
 will be provided to install the missing commands.
 
+
+Add `brew` tap for crossbuilding windows binaroes on MacOSX:
+
+```
+brew tap xamarin/xamarin-android-windeps
+```
+
+Details can be found: https://github.com/xamarin/homebrew-xamarin-android-windeps
+
+Otherwise, if building `xamarin-android` for the first time users might experience following error:
+
+```
+Checking xamarin/xamarin-android-windeps/mingw-zlib Error: No available formula with the name "xamarin/xamarin-android-windeps/mingw-zlib" 
+Please tap it and then try again: brew tap xamarin/xamarin-android-windeps
+```
+
 ### Brew Programs
 
 Suggested `brew install` commands:
 
+    brew install git
```

Comment:
This won't hurt, but `xaprepare` should take care of the dependency on its  own.